test/unit/tasks/emergency_banner_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'test_helper'
+require_relative '../../../lib/emergency_banner/deploy'
+require_relative '../../../lib/emergency_banner/remove'
+
+describe 'emergency_banner:deploy' do
```

Comment:
The structure of the tests in rails should mimic the structure of the code, so really, I should move these tests into `test/tasks`.  I'll do that and fixup. Thanks!